stade/core/templates/data.html

```diff
@@ -537,9 +552,9 @@ <h3>Citing 2019 datasets:</h3>
           Clínic de Barcelona</p>
         <p>HAM10000 Dataset: (c) by ViDIR Group, Department of
           Dermatology, Medical University of Vienna;
-          https://doi.org/10.1038/sdata.2018.161</p>
-        <p>MSK Dataset: (c) Anonymous; https://arxiv.org/abs/1710.05006 ;
-          https://arxiv.org/abs/1902.03368</p>
+          <a href="https://doi.org/10.1038/sdata.2018.161">https://doi.org/10.1038/sdata.2018.161</a></p>
+        <p>MSK Dataset: (c) Anonymous; <a href="https://arxiv.org/abs/1710.05006">https://arxiv.org/abs/1710.05006</a>;
+          <a href="https://arxiv.org/abs/1902.03368">https://arxiv.org/abs/1902.03368</a></p>
       </blockquote>
       When referencing this dataset in your own manuscripts
       and publications, please use the following full citations:
```

Comment:
There are some smart quotes on this page. Could you replace them all with straight quotes, to make copy-paste citation more friendly?